marquez_client/client.py

```diff
@@ -29,10 +29,15 @@
 _USER_AGENT = f'marquez-python/{VERSION}'
 _HEADERS = {'User-Agent': _USER_AGENT}
 
+_NAMESPACE_LENGTH_LIMIT = 1024
```

Comment:
_Minor:_ All names will be <= `1024`, so maybe define a single constant?

```python
_NAME = 1024
.
.
if len(namespace_name) > _NAME:
  raise ValueError('namespace_name length is {0}, must be <= {1}',
    len(namespace_name), _NAME)

# check job name using _NAME
# etc
```

Reply:
Isn't NAME a bit misleading? _NAME_LENGTH_LIMIT is clearer what it is.

Reply:
Yep, good point! What I _really_ wanted to type was `_MAX_NAME_LENGTH`.

Reply:
done.
`_check_name_length`